test/login/components/SearchHelpSignIn.unit.spec.jsx

```diff
@@ -1,21 +1,33 @@
 import React from 'react';
 import SkinDeep from 'skin-deep';
 import { expect } from 'chai';
+import _ from 'lodash';
 
 import { SearchHelpSignIn } from '../../../src/js/login/components/SearchHelpSignIn.jsx';
 
+const defaultLoginProps = {
+  currentlyLoggedIn: false,
+  loginUrl: '',
+  utilitiesMenuIsOpen: {
+    account: false,
+    help: false,
+    search: false
+  }
+};
+
+const defaultProfileProps = {
+  email: 'fake@aol.com',
+  userFullName: {
+    first: 'Sharon'
+  }
+};
+
+function buildProps(defaultProps, props = {}) {
+  return _.merge({}, defaultProps, props);
```

Comment:
@omgitsbillryan @U-DON I believe this doc is auto-generated from the lodash API docs and might be a bit more comprehensive: https://gist.github.com/jfmengels/6b973b69c491375117dc